### DISCUSSION TITLE
Miles, not km please

### DISCUSSION THREAD

**Discussion by dsnowdy** [teslamate-org/teslamate#3588] [Q&A] [answered]
I rebuilt my install and am getting km instead of miles. What's the easy fix? 🙃

**Comment by cwanja** [comment, marked as answer]
Check your settings.

**Comment by dsnowdy** [comment]
Thanks.  Took me a moment to recall where that was!
D

**Comment by doronkatz** [comment]
Where in settings? I am new and was searching as well. Saw suggestions to create sensor templates etc and other stuff.
